src/main/java/seedu/address/model/person/ClientStatus.java

```diff
@@ -0,0 +1,86 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's client status in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidPhone(String)}
+ */
+public class ClientStatus {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Client status must be either “active”, “unresponsive”, “potential”, or “old”(case sensitive).";
```

Comment:
Let's try to keep it case-insensitive. Simplest way is to just do `userinput.toLowerCase()` when taking the value from the user.